.github/scripts/check-status.py

```diff
@@ -1,59 +1,22 @@
 #!/usr/bin/env python3
 
-import re
-from os import environ, path
-from github import Github
-from stdm import get_latest_artifact_url
+from sys import argv
+from pathlib import Path
+from re import compile as re_compile
 
+PACKAGE_RE = re_compile("symbiflow-arch-defs-([a-zA-Z0-9_-]+)-([a-z0-9])")
 
-def check_status():
-    gh_ref = environ['GITHUB_REPOSITORY']
-    gh_sha = environ['INPUT_SHA']
-
-    MAIN_CI = "Architecture Definitions"
-
-    print('Getting status of %s @ %s...' % (gh_ref, gh_sha))
-
-    status = Github(
-        environ['INPUT_TOKEN']
-    ).get_repo(gh_ref).get_commit(sha=gh_sha).get_combined_status()
-
-    for item in status.statuses:
-        print('· %s: %s' % (item.context, item.state))
-
-    if status.state != 'success':
-        print('Status not successful. Skipping...')
-        exit(1)
-
-    if not any([item.context == MAIN_CI for item in status.statuses]):
-        print('Main CI has not completed. Skipping...')
-        exit(1)
-
-    artifacts, _ = get_latest_artifact_url()
-
-    PACKAGE_RE = re.compile("symbiflow-arch-defs-([a-zA-Z0-9_-]+)-([a-z0-9])")
-
-    for artifact in artifacts:
-        name = artifact["name"].split(".")[0]
-        url = artifact["url"]
-
-        m = PACKAGE_RE.match(name)
-        assert m, "Package name not recognized! {}".format(name)
+with (Path(__file__).parent.parent.parent / 'packages.list').open('r') as rptr:
```

Comment:
Where is the `packages.list` file coming from? I cannot find it checked-in, so maybe it is present only in CI. Maybe it is worth adding a comment on what it is and what it contains.

Reply:
It's created in CI, indeed: https://github.com/SymbiFlow/f4pga-arch-defs/pull/2713/files#diff-f88e7ee60a80885f57e6c7794c2dd3d3604ffed08c7c2bde856b1669186e8444R92. Eventually, I think we can merge steps 'Upload to Google Cloud Storage (GCS) bucket', 'Check Status' and 'Update latest URL files' into a single script. See https://github.com/SymbiFlow/f4pga-arch-defs/pull/2723/files#diff-f88e7ee60a80885f57e6c7794c2dd3d3604ffed08c7c2bde856b1669186e8444.